functions.php

```diff
@@ -317,88 +323,91 @@ function wp_autoupdates_plugins_bulk_actions( $actions ) {
  * Handles auto-updates enabling for plugins.
  */
 function wp_autoupdates_plugins_enabler() {
-	$action = isset( $_GET['action'] ) && ! empty( esc_html( $_GET['action'] ) ) ? wp_unslash( esc_html( $_GET['action'] ) ) : '';
-	if ( 'autoupdate' === $action ) {
-		if ( ! current_user_can( 'update_plugins' ) || ! wp_autoupdates_is_plugins_auto_update_enabled() ) {
-			wp_die( __( 'Sorry, you are not allowed to enable plugins automatic updates.', 'wp-autoupdates' ) );
-		}
+	if ( ! ( isset( $_GET['action'] ) && 'toggle-auto-update' === $_GET['action'] ) ) {
```

Comment:
Same as https://github.com/WordPress/wp-autoupdates/pull/119/commits/a7abdcd51734c9a3f54117c84d59ffc791c98acd#r421918954.

"Toggle" is not suitable, the action has to be explicit, either "enable" or "disable".